packages/reports/addon/components/dimension-selector.js

```diff
@@ -204,17 +200,11 @@ export default class DimensionSelector extends Component {
     target && target.focus(); // firefox does not focus a button on click in MacOS specifically
     const type = item.dateTimeDimension ? 'TimeGrain' : 'Dimension';
 
-    let actionHandler;
-
     if (type === 'TimeGrain') {
```

Comment:
We could change this check here to just be `item.dateTimeDimension `

Reply:
I dont think thats the case, I changed it as a test, and then all of the timegrain tests start failing

Reply:
Or do you mean just making the conditional if(item.dateTimeDimension) {

Reply:
Yea just update the conditional logic to take out the extra step